### DISCUSSION TITLE
Forced Device Reboot / Update Loop

### DISCUSSION THREAD

**Discussion by ORG-MBD** [Weatherlights/Winget-AutoUpdate-Intune#36] [Q&A]
Hello,

I've enabled AutoUpdate on User Logon for everyone, so WAU runs when somebody logs into the Device. Now I have the situation where some Devices are rebooting 15 Minutes after logon. Is there some way to suppress the Reboot ?

I've searched for some ADMX Policy but couldn't find anything. Also some Apps seems to be updating everytime to lets say V.1 -> V1.1 on Logon and on the next login its starting the same update from Ver.  V.1 -> V1.1.

Hope someone can help me :)

**Comment by Weatherlights** [comment]
Can you provide the logfiles as explained here: https://github.com/Weatherlights/Winget-AutoUpdate-Intune/wiki/FAQ#are-there-log-files

You maybe should start using a whitelist and only update stuff that does not result into reboots. There are some apps in Winget that cause funny things to happen and you should test all apps you approve to be updated with winget.
